hack/deploy_cincinnati.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# The script serves as an example of how Cincinnati can be run on an OpenShift cluster. 
+
+# Create a dummy secret as a workaround to not having real secrets
+oc create secret generic cincinnati-credentials --from-literal=""
+
+
+# Create a new namespace/project for Cincinnati
+oc new-project cincinnati
+
+
+# Apply oc template
+oc new-app -f dist/openshift/cincinnati.yaml \
+  -p RUST_BACKTRACE="1" \
+  -p IMAGE_TAG="90efacd" \
+  -p GB_MEMORY_REQUEST=256Mi \
```

Comment:
Should we recommend overriding default requests / limits?

Reply:
good point, I would remove these

Reply:
Done by f0b1bd460325fa2d11b1cdca9574a278cf66580e.